etna/transforms/decomposition/stl.py

```diff
@@ -194,7 +194,9 @@ def __init__(
             2. "holt": ``ETSModel(data, trend='add')``
 
             Custom model should be a subclass of :py:class:`statsmodels.tsa.base.tsa_model.TimeSeriesModel`
-            and have method ``get_prediction`` (not just ``predict``)
+            and have method ``get_prediction`` (not just ``predict``).
+
+            During ``save`` pickle is used and model should support saving and loading with pickle.
```

Comment:
Why do we need to specify this?

Reply:
I think there is a possibility that some users might give some model that doesn't support pickling.